cmd/pgsql.go

```diff
@@ -10,6 +10,7 @@ import (
 var (
 	PgSQLConsoleCommand = cli.Command{
 		Name:     "pgsql-console",
+		Aliases: []string{"psql-console","postgresql-console"},
```

Comment:
Haha I didn't about this attribute, look at how I did it for another alias 😄 

https://github.com/Scalingo/cli/blob/5b11bd7de1a97f8c4c4ce70e98e8c023571af67d/cmd/deployments.go#L36-L44

What I prefer with my solution, is that the output of `scalingo help` does not mention this alias. It is some kind of typo helper and I don't really want to advertise it. 

And in this case my opinion I have mixed feeling whether I want to advertise about these aliases in `scalingo help`. Do you have an opinion on this?

Reply:
I truly don't have any opinion on this matter. As you like.

Reply:
I agree with Etienne. In addition, it overloads the help command i think.

Afterwards, if we could display these aliases in the help of the command directly rather than in the `scalingo help`, that would be a plus, but that's a detail

Reply:
On my side I was confused the first time I saw the "Étienne's" solution in the `deployment-` autocomplete because it first didn't give me the impression that it was an alias but another command.
So I am interested in the other solution, maybe this one can be better.
Personally I find it cleaner this:
```
   Databases:                                                                                                          
     redis-console                                    Run an interactive console with your Redis addon                 
     mongo-console                                    Run an interactive console with your MongoDB addon               
     mysql-console                                    Run an interactive console with your MySQL addon                 
     pgsql-console, psql-console, postgresql-console  Run an interactive console with your PostgreSQL addon            
     influxdb-console                                 Run an interactive console with your InfluxDB addon              
```
instead of duplication of line like:
```
   Deployment:                                                                                                         
     deployments              List app deployments                                                                     
     deployment-logs          View deployment logs                                                                     
     deployment-follow        Follow deployment event stream                                                           
     deploy                   Trigger a deployment by archive                                                          
     deployment-delete-cache  Reset deployment cache                                                                   
     deployment-cache-delete  Reset deployment cache                                                                   
```

Reply:
OK I think I eventually agree with Pierre an Léo. Let's use this `Aliases` attribute! Can you also update the `deployment-cache-delete` so that it's just an alias to `deployment-delete-cache`? Thanks 🙏🏾

Reply:
Same for `self` and `whoami` ?
And should I keep the duplicated code or not (or maybe comment it) ?

Reply:
Yes, please do the same for `self` and `woami`. Keep `self` and make `whoami` an alias of `self`.

You can remove the duplicated code.

Reply:
I'm on it.

Reply:
@curzolapierre
The alias inherits from it's original command for the autocompletion of the flags.